internal/pkg/manifest/workload.go

```diff
@@ -78,27 +76,8 @@ type Workload struct {
 
 // OverrideRule holds the manifest overriding rule for CloudFormation template.
 type OverrideRule struct {
-	TaskDefOverrideRules []TaskDefinitionOverrideRule `yaml:"taskdef_override"`
-}
-
-// TaskDefinitionOverrideRule holds the overriding rule for ECS Task Definition.
-type TaskDefinitionOverrideRule string
-
-// Parse parses the manifest ECS Task Definition overriding rule into override.Rule.
-func (r TaskDefinitionOverrideRule) Parse() (*override.Rule, error) {
-	// Make sure the overriding rule is in a map format.
-	// For example: "ContainerDefinitions[0].Ulimits[-].HardLimit: !Ref ParamName"
-	if len(strings.Split(string(r), ":")) != 2 {
-		return nil, fmt.Errorf("invalid overriding rule %s", r)
-	}
-	var node yaml.Node
-	if err := yaml.Unmarshal([]byte(r), &node); err != nil {
-		return nil, fmt.Errorf("unmarshal task definition overriding rule %s: %w", r, err)
-	}
-	return &override.Rule{
-		Path:  node.Content[0].Content[0].Value,
-		Value: node.Content[0].Content[1],
-	}, nil
+	Path  string    `yaml:"path"`
```

Comment:
look at all that sweet deleted code 😎

Reply:
Do we need `Path *string` and `Value *yaml.Node`?

Reply:
`Value *yaml.Node` won't work...and I don't think empty string would be a valid value for `Path`

Reply:
If `path` isn't specified in manifest, wont it get initialized as `""`? In that case, how do we tell this from an actual invalid input where the user specifically put `""` for `path`?

Reply:
https://github.com/aws/copilot-cli/pull/2743/files#diff-05339d6eda8f6845d17d754ca2a2cb1b58ebd84ad763d1f9b31a961188eb2a4dR48 if it isn't specified then `taskdef_overrides` length would be 0 and we wouldn't even loop through it. Unless they have
```yaml
taskdef_override:
  - Value: foo
```
then we'll error out.